app/components/visualizer/main-viz/templates/chart.jsx

```diff
@@ -0,0 +1,48 @@
+import D3Chart from "./D3Chart.js"
+
+class Chart extends React.Component {
+  componentDidMount() {
+    let el = React.findDOMNode(this);
+    this.chart = new D3Chart(el, {
+      width: "100%",
```

Comment:
I actually did that, then wasn't sure what commits were yours and now my branch is in some weird limbo state where I can't save anything :/